src/iris/ui/auth/ldap.py

```diff
@@ -11,7 +11,7 @@ def __init__(self, config):
         root = os.path.abspath('./')
         self.ldap_url = config['auth']['ldap_url']
         self.cert_path = os.path.join(root, config['auth']['ldap_cert_path'])
-        self.user_suffix = config['auth']['ldap_user_suffix']
+        self.user_search_string = config['auth']['ldap_user_search_string']
```

Comment:
Can you alter the config.yaml example files to account for this change?

Reply:
@dwang159 There is no `config.yam` example in project, and the `config.dev.yaml` does not  have any ldap login config in it.

Reply:
Hm, you're right. I was thinking about the linkedin/oncall repo, sorry.

Reply:
Can you make this support both config options? (So we and other people can avoid changing our configs)

Eg if you only specify `ldap_user_suffix`, default `self.user_search_string` to `'%%s%s' % config['auth']['ldap_user_suffix']` to preserve the old behavior